### ISSUE TITLE
Some Feedback

### ISSUE THREAD

**Issue by zoffixznet** [Raku/raku.org#35]
Fished out from https://news.ycombinator.com/item?id=10331008
- Would be nice to make the intro copy more informative. Like: why would I want to use it as a developer? What does it look like compared to my favorite language?
- Really like the examples, especially since I haven't done much with Perl 6. However, like Perl 5, it still looks like parseable line noise to the untrained eye. Would be nice to have some better explanations to go along with the code. Maybe not right on the homepage, but at least linked into it.
- Would like to see the "For Newcomers" links on the "Documentation" page on the homepage.
- This Issue: https://github.com/perl6/perl6.org/issues/36
- The page looks fine. I think the "Fun" and "Whatever" pages could be merged. The front page should probably explain more why someone would want to use the language.
- That said though, none of this is going to matter unless there's a good tutorial. Nothing currently available is cutting it as a tutorial.

**Comment by jnthn** [comment]
Yes, the intro text really should be more focused on Perl 6 the language,
not Perl 6 the project, at this point.

For examples, I'd probably focus on ones that are pretty clean and show off
useful rather than cute things. The first one now is pretty good. I'd maybe
got for an OO one, something doing some neat list-processing, and maybe an
async one with await or with react/whenever.

**Comment by stmuk** [comment]
I wonder if some of the copy from

http://www.perlfoundation.org/perl6/index.cgi?perl_6_executive_summary

might be useful.

Although it needs editing down to maybe 5 or 6 bullet points, spell
correction and updating with probably more emphasis on newer features such
as async and parallel programming

eg.

   - Perl 6's many new built-in features greatly advance the Perl tradition
      of highly-capable, feature-rich programming. Here are some of those
      features:
         - Multimethods, coroutines, continuations, currying, signatures,
         captures, exceptions
         - Powerful yet convenient object-oriented programming, generics,
         roles
         - Functional programming primitives, {lazy and eager} evaluation,
         junctions, autothreading, hyperoperators (vector operators)
         - Definable grammars for {pattern matching and generalized string
         processing}, with many other powerful extensions
         - Advanced introspection and meta-programming
         - {Extensible and overridable} {Perl 6 primitives and Perl 6
         grammar}, language-level macros
         - Module aliasing and versioning, optional static/gradual typing
         - Full Unicode processing support
         - Garbage collection
         - Greatly-improved foreign function interface
         - Optional Tyoing System






On 5 October 2015 at 16:05, Jonathan Worthington <notifications@github.com>
wrote:

> Yes, the intro text really should be more focused on Perl 6 the language,
> not Perl 6 the project, at this point.
>
> For examples, I'd probably focus on ones that are pretty clean and show off
> useful rather than cute things. The first one now is pretty good. I'd maybe
> got for an OO one, something doing some neat list-processing, and maybe an
> async one with await or with react/whenever.
>
> On Mon, Oct 5, 2015 at 5:03 PM, Zoffix Znet <notifications@github.com>
> wrote:
>
> > Fished out from https://news.ycombinator.com/item?id=10331008
> >
> > Would be nice to make the intro copy more informative. Like: why would I
> > want to use it as a developer? What does it look like compared to my
> > favorite language?
> >
> > Really like the examples, especially since I haven't done much with Perl
> > 6. However, like Perl 5, it still looks like parseable line noise to the
> > untrained eye. Would be nice to have some better explanations to go along
> > with the code. Maybe not right on the homepage, but at least linked into
> it.
> >
> > Would like to see the "For Newcomers" links on the "Documentation" page
> on
> > the homepage.
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <https://github.com/perl6/perl6.org/issues/35>.
>
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/perl6/perl6.org/issues/35#issuecomment-145562188>.
>



-- 
4096R/EA75174B Steve Mynott <steve.mynott@gmail.com>

**Comment by coke** [comment]
Closing as resolved given the latest redesign.
